var/spack/repos/builtin/packages/py-cfgrib/package.py

```diff
@@ -13,12 +13,27 @@ class PyCfgrib(PythonPackage):
     homepage = "https://github.com/ecmwf/cfgrib"
     pypi     = "cfgrib/cfgrib-0.9.8.5.tar.gz"
 
+    version('0.9.9.0', sha256='6ff0227df9c5ee34aa7d6ab1f7af3fbe6838523a8a9891c74040b419b03ad289')
     version('0.9.8.5', sha256='07c224d7ac823a1df5738b96b9d3621515538f51f67e55044f9cc8ec1668e1bd')
 
+    # Warning: can create infinite dependency loop with xarray+io ^cfgrib+xarray
```

Comment:
Is it possible to define a conflict or something that prevents this?

Reply:
Not sure how the new concretizer would handle this, maybe @alalazo would know.